src/lib/db.ts

```diff
@@ -315,15 +331,34 @@ export class DB {
           if (!document || !metadata) {
             throw new Error(`Unable to open document ${documentKey}`);
           }
+
+          // TODO: When the 4-up code is merged, we need a flag to disable this monitoring when we're
```

Comment:
I think we do want to monitor the other group documents so that they show in real time in the 4 up view right?

Reply:
We'll definitely want to monitor group documents in Firebase - but I'm not sure we have any reason for watching their models locally since we'll never change them, right?